Description: set a static version string rather than discern it from git
--- a/meson.build
+++ b/meson.build
@@ -17,6 +17,7 @@ add_project_arguments(
 
 conf = configuration_data()
 conf.set_quoted('PACKAGE_NAME', meson.project_name())
+conf.set('VCS_TAG', meson.project_version())
 conf.set('_GNU_SOURCE', 1, description : 'Enable GNU extensions on systems that have them.')
 
 build_arping = get_option('BUILD_ARPING')
@@ -207,10 +208,10 @@ foreach h : [
 	endif
 endforeach
 
-git_version_h = vcs_tag(
+git_version_h = configure_file(
 	input : 'git-version.h.meson',
 	output : 'git-version.h',
-	fallback : meson.project_version()
+	configuration: conf
 )
 
 config_h = configure_file(
